mflowgen/Tile_PE/constraints/constraints.tcl

```diff
@@ -32,15 +32,18 @@ set_load -pin_load $ADK_TYPICAL_ON_CHIP_LOAD [all_outputs]
 set_driving_cell -no_design_rule \
   -lib_cell $ADK_DRIVING_CELL [all_inputs]
 
-# set_input_delay constraints for input ports
-#
-# - make this non-zero to avoid hold buffers on input-registered designs
-
-set_input_delay -clock ${clock_name} [expr ${dc_clock_period}/2.0] [all_inputs]
-
-# set_output_delay constraints for output ports
-
-set_output_delay -clock ${clock_name} 0 [all_outputs]
+set_output_delay -clock ${clock_name} [expr ${dc_clock_period}*0.8] [get_ports hi]
+set_output_delay -clock ${clock_name} [expr ${dc_clock_period}*0.8] [get_ports lo]
```

Comment:
FYI hi and lo are constant outputs, so this isn't necessary. These should be false paths.